test/unit/org/apache/cassandra/sensors/SensorsInternodeTest.java

```diff
@@ -188,6 +196,48 @@ public void testInternodeSensorsForCounterMutation()
         testInternodeSensors(request, handler, ImmutableSet.of(context));
     }
 
+    @Test
+    public void testInternodeSensorsForLWTPrepare()
```

Comment:
I'd recommend adding tests with CQL CAS queries.

Reply:
I tired in the past and failed tbh :\ - it is far from trivial. Unit tests won't cut it because we need multi-writer setup to exchange LWT messages, but I tried with the distributed tests (see org.apache.cassandra.distributed.test.CASTest for an example) and failed to intercept message (this is an mocked messaging service but doesn't have a public API). 

So far we've been testing sensors by invoking verb handlers in units tests. If we go the distributed tets way, we really need to capture this effort separately.

Reply:
I see. I've figured that we have integration tests in CNDB so def not a must.